etc/ci/upload_nightly.sh

```diff
@@ -25,9 +25,8 @@ upload() {
     s3cmd cp "${package_upload_path}" "${nightly_upload_dir}/servo-latest.${3}"
 }
 
-
```

Comment:
Keep this line to avoid diff noise and fit with the double newline between functions style.